.github/workflows/documenter.yml

```
@@ -17,7 +17,7 @@ jobs:
           version: 1.3.361
       - uses: julia-actions/setup-julia@latest
         with:
-          version: 1.9
+          version: 1.10
```

```suggestion
          version: "1.10"
```
Without `""` YAML considers it to be a floating point number equal to 1.1.

---

Ah yes debugged for quite a while the same issue on Manopt and was not careful enough yesterday.